src/Stratis.Bitcoin/Connection/ConnectionManager.cs

```diff
@@ -234,8 +234,9 @@ private void AddComponentStats(StringBuilder builder)
             {
                 var chainHeadersBehavior = peer.Behavior<ConsensusManagerBehavior>();
 
-                string peerHeights = $"(r/s):{(chainHeadersBehavior.BestReceivedTip != null ? chainHeadersBehavior.BestReceivedTip.Height.ToString() : peer.PeerVersion?.StartHeight + "*" ?? "-")}";
+                string peerHeights = $"(r/s/c):{(chainHeadersBehavior.BestReceivedTip != null ? chainHeadersBehavior.BestReceivedTip.Height.ToString() : peer.PeerVersion?.StartHeight + "*" ?? "-")}";
```

Comment:
`?? "-"` is unreachable
same on the line below

Reply:
I preserved the previous code. If you are sure that PeerVersion can never be null than we can remove it

Reply:
it maybe can. but even if it is `?? "-"` is still unreachable

Reply:
I agree it will never be reached because you always have the `+ "*"`  I will fix it

Reply:
ah ok I see what you mean